pythonfile/first.py

```diff
@@ -1,4 +1,9 @@
 from pymongo import MongoClient
+import elastic
+import time
+
+time.sleep(5) #wait for initialisation of elasticsearch and mongodb...
```

Comment:
Okay for testing, but this cannot be the solution in the long term.
We will have to wait for the sockets to become available.
Please add another `FIXME` here, so we don't forget that.